### ISSUE TITLE
Roman Liburna

### ISSUE THREAD

**Issue by Spheniscine** [Thalassicus/cep-bnw#273]
Roman unique Trireme, Liburna, looks identical to the Bireme and has the same unit flag, making it hard to tell them apart in fleets that are a mixture of the two.

**Comment by GrantSP** [comment]
Two options here, either we find another icon from the core files or we create our own. 
The first is probably not a way to go as it should have been done already.

**Comment by Spheniscine** [comment]
One partial solution might be to use the Trireme (vanilla Quinquereme) graphics instead; not ideal to have a unique unit look like its base type (as one can forget that it's unique), but better than looking like another class, especially one that's often mixed with in fleets.

**Comment by GrantSP** [comment]
I'll have a look tomorrow. If the trireme is good we can make the change.

**Comment by GrantSP** [comment]
I remember this now. When we first made this unit I used another graphic to avoid the duplication issue you mention. @Thalassicus preferred the current graphic and made the change. Both versions had the same issue, it just depends on where you focus as to which one you prefer.

End result is we are limited with the vanilla graphics as to what to use. I tried looking online for some more graphics that might serve the purpose but came up empty-handed.
Unless some new graphics emerge we will stay with what we have.
